x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -81,6 +85,24 @@ export const graphableProcesses = createSelector(
   }
 );
 
+/**
+ * Process events that will be displayed as terminated.
+ */
+export const terminatedProcesses = createSelector(
+  ({ results }: DataState) => results,
+  function (results: DataState['results']) {
+    return new Set(
+      results.filter(isTerminatedProcess).map((terminatedEvent) => {
+        if (event.isLegacyEvent(terminatedEvent)) {
+          return terminatedEvent.endgame.unique_pid;
```

Comment:
Maybe this should use `entityId` here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/common/endpoint/models/event.ts#L42

Reply:
or https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/resolver/models/process_event.ts#L73